packages/components/src/components/hds/accordion/index.ts

```diff
@@ -26,7 +26,7 @@ interface HdsAccordionSignature {
     default: [
       {
         Item?: ComponentLike<HdsAccordionItemSignature>;
-      },
+      }
```

Comment:
I merged in the latest code from main and got all these linting changes from the new rule for including/excluding commas. Need to figure out how to fix...